### PR TITLE
Close popups on any click

### PR DESCRIPTION
**Pull request by stephanfowler**
@sndrs @robertberry

### CODE REVIEW

**Comment by robertberry** [comment]
Nice work :+1:

**Comment by paperboyo** [comment]
:gem: Thank you!

**Comment by piuccio** [comment]
great idea, but a feature now depends on an analytic module. Probably wouldn't work if you click on a toggle that doesn't contain a data-link-name.

**Comment by stephanfowler** [comment]
@piuccio we [use clickstream](https://github.com/guardian/frontend/search?q=clickstream) for various toggle behaviours already. It's our single emitter of click info.

**Comment by piuccio** [comment]
fair enough :+1:

File: static/src/javascripts/projects/common/modules/ui/toggles.js [hunk 1]
```diff
@@ -13,8 +15,9 @@ define([
             readyClass = 'js-toggle-ready';
 
         this.init = function () {
-            controls = document.body.querySelectorAll('[data-toggle]');
-            Array.prototype.forEach.call(controls, function (control) {
+            controls = Array.prototype.slice.call(document.body.querySelectorAll('[data-toggle]'), 0);
```
Comment (rich-nguyen) [inline static/src/javascripts/projects/common/modules/ui/toggles.js]: do you need `0`?

Comment (stephanfowler) [inline static/src/javascripts/projects/common/modules/ui/toggles.js]: No. I don't.

**Pull request merged**